.pre-commit-hooks.yaml

```
@@ -0,0 +1,8 @@
+- id: spellchecker-cli
+  name: spellchecker
+  entry: spellchecker
+  language: node
+  types: [markdown]
```

Would it make sense to also spellcheck other kinds of plain text files here? Perhaps:

```suggestion
  types: [text]
```

Or

```suggestion
  types_or: [text, markdown]
```

If `markdown` doesn't count as a subtype of `text`.

---

https://github.com/pre-commit/identify/blob/main/identify/extensions.py#L141 
I think markdown is considered text! 

I think if it's widened to `text`, then the `--files` flag would have to be present if anyone is looking for specific types. My team's usecase is for `markdown` only so we'd probably tune it to `--files "**/*.md`. 

FWIW, Turning to `text` in my local test just now (`pre-commit try-repo ../spellchecker-cli/ spellchecker-cli --all-files`) grabbed `.yaml`, '.ipynb`, `.ignore`-like files, and more.

---

I was just going to ask about jupyter notebooks and comments (docstring) in python scripts... 🤣
Does it works for python too?